src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -1,23 +1,190 @@
 import React from 'react';
 import {RouteComponentProps} from 'react-router/lib/Router';
+import styled from '@emotion/styled';
 
-import {t} from 'app/locale';
+import {Panel, PanelBody, PanelHeader, PanelAlert} from 'app/components/panels';
+import {t, tct} from 'app/locale';
 import AsyncComponent from 'app/components/asyncComponent';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import {Organization} from 'app/types';
+import ExternalLink from 'app/components/links/externalLink';
+import space from 'app/styles/space';
+import Button from 'app/components/button';
+import {IconDelete} from 'app/icons';
+import Tooltip from 'app/components/tooltip';
+
+import Time from './time';
+
+const RELAY_DOCS_LINK = 'https://getsentry.github.io/relay/';
+
+type Relay = {
+  public_key: string;
+  name: string;
+  created: string;
+  first_seen: string;
+  last_seen: string | null;
+  last_modified?: string;
+  description?: string;
+};
 
 type Props = {
   organization: Organization;
 } & RouteComponentProps<{orgId: string}, {}>;
 
-class Relays extends AsyncComponent<Props> {
+type State = AsyncComponent['state'] & {
+  relays: Array<Relay>;
+};
+
+const relaysMock: Array<Relay> = [
```

Comment:
Is this view hidden behind a feature flag?

Reply:
yes :)